kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala

```diff
@@ -78,34 +75,6 @@ class FlinkProcessBuilder(
 
   override protected def commands: Array[String] = Array(executable)
 
-  override def killApplication(clue: Either[String, String]): String = clue match {
-    case Left(_) => ""
-    case Right(line) => killApplicationByLog(line)
-  }
-
-  def killApplicationByLog(line: String = lastRowsOfLog.toArray.mkString("\n")): String = {
-    "Job ID: .*".r.findFirstIn(line) match {
```

Comment:
this is an invalid implementation so I didn't move it somewhere but deleted it directly.

Reply:
Because of the invalid implementations, I would like to open a pull request to support this. You could delete it directly.

Reply:
thanks @SteNicholas